roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -1,5 +1,6 @@
 """Check that required Docker images are available."""
 
+from pipes import quote
```

Comment:
nit: **Not worth slowing down the merge now**, but this is really ``shlex.quote``. At a future time it would make reading the code easier if this was:

```python
import shlex
```

and later used as

```python
shlex.quote(..)
```

Reply:
roger, i'll stick that in my tech debt file as this is definitely not the last time this check will be revisited (probably this week)